xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -543,7 +544,7 @@ bool CGUIControlButtonSetting::OnClick()
     }
     else if (controlFormat == "path")
       SetValid(GetPath((CSettingPath *)m_pSetting));
-    else if (controlFormat == "action")
+    else if (controlFormat == "action" || controlFormat == "infoaction")
```

Comment:
Is handled exactly the same as a normal button with action only, but on line 618 the label becomes continuously updated with it. Don't want to do this for everything, thats why separate name.